app/controllers/scim_rails/scim_groups_controller.rb

```diff
@@ -45,5 +73,67 @@ def patch_update
 
     private
 
+    def put_error_check
+      members = params["members"]
+
+      raise ScimRails::ExceptionHandler::InvalidPutMembers unless (members.is_a?(Array) && array_of_hashes?(members))
+
+      member_ids = members.map{ |member| member["value"] }
+
+      member_ids.each do |id|
+        @company.public_send(ScimRails.config.scim_users_scope).find(id)
+      end
+        
+      return if put_active_param.nil?
+
+      case put_active_param
+      when true, "true", 1
+        
+      when false, "false", 0
+        
+      else
+        raise ScimRails::ExceptionHandler::InvalidActiveParam
+      end
+    end
+
+    def array_of_hashes?(array)
+      array.each do |element|
+        return false unless element.is_a?(Hash)
+      end
+
+      return true
```

Comment:
🌵 `return` doesn't do anything if it's already the last line in a method.
The only reason to use `return` in ruby is to *break* the current flow before its natural end.
```suggestion
      true
```